### PR TITLE
chore(doc): update slim readme for Subscription Agreement

### PR DESCRIPTION
**Pull request by ericsyh**
*(If this PR fixes a github issue, please add `Fixes #<xyz>`.)*

Fixes #<xyz>

*(or if this PR is one task of a github issue, please add `Master Issue: #<xyz>` to link to the master issue.)*

Master Issue: #<xyz>

### Motivation

Follow with https://github.com/streamnative/charts/pull/1015

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [x] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

Check the box below.

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [ ] `no-need-doc` 
  
  (Please explain why)
  
- [x] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

File: charts/sn-platform-slim/README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # What is StreamNative Platform Slim
 
+> NOTE: Installing the StreamNative Platform you agree to and are in compliance with the [StreamNative Cloud Subscription Agreement](https://streamnative.io/cloud-terms-and-conditions).  Please [contact StreamNative](https://streamnative.io/contact) to learn more or request access to a trial license.
```
Comment (Huanli-Meng) [inline charts/sn-platform-slim/README.md:3]:
```suggestion
> **Note**
>
> Installing StreamNative Platform indicates that you agree to and are in compliance with the [StreamNative Cloud Subscription Agreement](https://streamnative.io/cloud-terms-and-conditions). To learn more or request access to a trial license, Please [contact StreamNative](https://streamnative.io/contact).
```

**Review by Huanli-Meng — APPROVED** [review]
Only one minor comment.

**Comment by ericsyh** [comment]
@Huanli-Meng PTAL again, thanks.

**Review by Huanli-Meng — APPROVED** [review]
LGTM

**Comment by ericsyh** [comment]
@tuteng @urfreespace @maxsxu @FushuWang @ciiiii PTAL

**Review by tuteng — APPROVED** [review]

**Pull request merged**